.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
Out of curiosity what's being configured in launch.json and settings.json?

Reply:
Oh, I didn't know those weren't git ignored. This is a VSCode configuration to debug Jest tests. It's actually very handy. You can use all the debugger tools while running tests.

[link to article for source](https://mattmazzola.medium.com/how-to-debug-jest-tests-with-vscode-48f003c7cb41)

Reply:
Awesome!